content/id/docs/reference/glossary/docker.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Docker
+id: docker
+date: 2018-04-12
+full_link: https://docs.docker.com/engine/
+short_description: >
+  Docker merupakan suatu teknologi perangkat lunak yang menyediakan virtualisasi pada level sistem operasi yang juga dikenal sebagai Container.
+aka:
+tags:
+- fundamental
+---
+Docker (secara spesifik, Docker Engine) merupakan suatu teknologi perangkat lunak yang menyediakan virtualisasi pada level sistem operasi yang juga dikenal sebagai {{< glossary_tooltip term_id="container" >}}.
+
+<!--more-->
+
+Docker menggunakan fitur isolasi sumber daya pada kernel Linux seperti cgroup dan _namespace_, dan [UnionFS](https://docs.docker.com/get-started/overview/#union-file-systems) seperti OverlayFS dan lainnya untuk memungkinkan masing-masing Container dijalankan pada satu instans Linux, menghindari beban tambahan (_overhead_) saat memulai dan menjalankan VM.
```

Comment:
Keep "instance", because "instans" is not official words, also "instan" have different meaning.
```suggestion
Docker menggunakan fitur isolasi sumber daya pada kernel Linux seperti cgroup dan _namespace_, dan [UnionFS](https://docs.docker.com/get-started/overview/#union-file-systems) seperti OverlayFS dan lainnya untuk memungkinkan masing-masing Container dijalankan pada satu _instance_ Linux, menghindari beban tambahan (_overhead_) saat memulai dan menjalankan VM.
```

Reply:
hmm, it's based on RSNI https://github.com/jk8s/sig-docs-id/blob/master/resources/RSNI-glossarium.pdf, don't we refer to this doc too for translation?